doc/src/vtr/server_mode/index.rst

```diff
@@ -0,0 +1,203 @@
+          count_adder_carry_p_cout[9].cout[0] (adder_carry)                          0.070     2.461
+          count_adder_carry_p_cout[10].cin[0] (adder_carry)                          0.053     2.514
+          count_adder_carry_p_cout[10].cout[0] (adder_carry)                         0.070     2.584
+          count_adder_carry_p_cout[11].cin[0] (adder_carry)                          0.043     2.627
+          count_adder_carry_p_cout[11].cout[0] (adder_carry)                         0.070     2.696
+          count_adder_carry_p_cout[12].cin[0] (adder_carry)                          0.053     2.749
+          count_adder_carry_p_cout[12].cout[0] (adder_carry)                         0.070     2.819
+          count_adder_carry_p_cout[13].cin[0] (adder_carry)                          0.043     2.862
+          count_adder_carry_p_cout[13].cout[0] (adder_carry)                         0.070     2.932
+          count_adder_carry_p_cout[14].cin[0] (adder_carry)                          0.053     2.985
+          count_adder_carry_p_cout[14].sumout[0] (adder_carry)                       0.040     3.025
+          count_dffsre_Q_D[13].in[0] (.names)                                        0.564     3.589
+          count_dffsre_Q_D[13].out[0] (.names)                                       0.228     3.818
+          count[13].D[0] (dffsre)                                                    0.000     3.818
+          data arrival time                                                                    3.818
+
+          clock clk (rise edge)                                                      0.000     0.000
+          clock source latency                                                       0.000     0.000
+          clk.inpad[0] (.input)                                                      0.000     0.000
+          count[13].C[0] (dffsre)                                                    0.715     0.715
+          clock uncertainty                                                          0.000     0.715
+          cell setup time                                                           -0.057     0.659
+          data required time                                                                   0.659
+          ------------------------------------------------------------------------------------------
+          data required time                                                                   0.659
+          data arrival time                                                                   -3.818
+          ------------------------------------------------------------------------------------------
+          slack (VIOLATED)                                                                    -3.159
+
+
+          #End of timing report
+          #RPT METADATA:
+          path_index/clock_launch_path_elements_num/arrival_path_elements_num
+          0/2/30
+      ",
+        "STATUS": "1"
```

Comment:
Not sure if it matters or not, but DATA and STATUS are in the opposite order in the figure. If this matters, I'd align the figure and example.

Reply:
The order of elements in telegram body is not matter, since it's json format.
The order of bytes is matter only for telegram header, which is fixed size structure.
Anyway i will update order of fields in json to be match.

Reply:
solved https://github.com/w0lek/vtr-verilog-to-routing/pull/11/commits/c27edfb212e81720adb8ccc1c9973efe977aa0a3
(will be updated later)